core/iwasm/common/wasm_runtime_common.c

```diff
@@ -3727,6 +3725,45 @@ wasm_runtime_invoke_native_raw(WASMExecEnv *exec_env, void *func_ptr,
                 break;
             }
             case VALUE_TYPE_I64:
+#if WASM_ENABLE_MEMORY64 != 0
```

Comment:
Should add `#if WASM_ENABLE_MEMORY64 == 0 ... #endif` in the above `case VALUE_TYPE_I32`